src/main/kotlin/io/kotlintest/KotlinTestEngine.kt

```diff
@@ -0,0 +1,220 @@
+package io.kotlintest
+
+import org.junit.platform.commons.JUnitException
+import org.junit.platform.commons.util.ReflectionUtils
+import org.junit.platform.engine.*
+import org.junit.platform.engine.discovery.PackageSelector
+import org.junit.platform.engine.support.hierarchical.EngineExecutionContext
+import java.lang.reflect.Modifier
+import java.util.*
+
+class KotlinTestContext(val executionRequest: ExecutionRequest) : EngineExecutionContext
```

Comment:
It seems that KotlinTestContext is never used. Is it used via reflection? If so, it should be documented.